src/editor/EditorManager.js

```diff
@@ -779,6 +778,25 @@ define(function (require, exports, module) {
         }
     }
 
+    function findProvider(editor,provider) {
+        var providers = (provider)?editProvider:docProvider,
```

Comment:
What is this code doing?  It's unclear why you're choosing one vs the other here based on `provider`.